loc/en.js

```diff
@@ -363,9 +363,9 @@ module.exports = {
       type: 'Transaction type',
       watchOnlyError: 'Watch only wallets cannot send transactions',
     },
-    warning: 'Warning:',
+    warning: 'Warning: ',
```

Comment:
I'm just wondering if it'd be better to remove whitespace from here and add it to the component in the place where it is displayed? Something like:
```
`${warning} ${warningGeneral}`
``` 

?

Reply:
No because it needs different styling